### ISSUE TITLE
.plot() doesn't display plot

### ISSUE THREAD

**Issue by ming-cui** [ModelOriented/DALEX#522]
I'm using JupyterLab on Gradient. Sample code, like `vi.plot(max_vars=5)`, can run but can't display the plot successfully. I'm not sure if JupyterLab or Gradient caused this issue. I've tried Deepnote and it works well. Thank you.

**Comment by hbaniecki** [comment]
Could you check if `plotly` works on Gradient in general? https://plotly.com/python/box-plots/

**Comment by ming-cui** [comment]
Thanks for your reply. No. Same issue with `plotly`. However, when I installed `plotly`, I got this message `WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv`. Could this be the cause? I'm not familiar with virtual env. Thanks again!
